gnomad_qc/v4/resources/annotations.py

```diff
@@ -253,3 +253,49 @@ def get_freq_comparison(version1, data_type1, version2, data_type2):
         ht_path = f"gs://gnomad/annotations/hail-0.2/ht/{data_type1}/{ht_path}"
 
     return TableResource(ht_path)
+
+
+def get_insilico_predictors(
+    version: str = CURRENT_VERSION,
+    predictor: str = "cadd",
+) -> VersionedTableResource:
+    """
+    Get the path to the in silico predictors TableResource for a specified release.
+
+    :param version: Version of annotation path to return.
+    :param predictor: One of the in silico predictors available in gnomAD v4, including cadd, revel, primate_ai, splice_ai, and pangolin.
+    :return: in silico predictor VersionedTableResource for gnomAD v4.
+    """
+    return VersionedTableResource(
+        CURRENT_VERSION,
+        {
+            version: TableResource(
+                path=f"gs://gnomad/v{version}/annotations/in_silico_predictors/gnomad.v{version}.{predictor}.grch38.ht"
+            )
+            for version in VERSIONS
+        },
+    )
+
+
+def get_vrs(
```

Comment:
"gs://gnomad-vrs-io-finals/" was partly used for v3 because of issues not being able to write to the typical gnomad buckets.
can you add something like this for get_vrs:
"param with_original_annotations: Whether to obtain just the Table of VRS annotations or, if it exists, the original input Table that was annotated with the VRS annotation added on"

and then drop the use of "gs://gnomad-vrs-io-finals/" in the script, so the main output of run_vrs will go to get_vrs() and the output of annotate_original will go to get_vrs(with_original_annotations=True)?

Reply:
nope, it's fine to keep it as in shouldn't be a huge ht